web-ng/src/custom-theme.scss

```diff
@@ -27,12 +27,17 @@
 // Available color palettes: https://material.io/design/color/
 // TODO: Use custom color theme slightly with more saturated colors used in
 // Ground Android app.
-$web-ng-primary: mat-palette($mat-green, 700, 600, 800);
-$web-ng-accent: mat-palette($mat-amber, 700, 600, 800);
-$web-ng-warn: mat-palette($mat-pink, 600, 500, 700);
+$primary: mat-palette($mat-green, 700, 600, 800);
+$accent: mat-palette($mat-amber, 700, 600, 800);
+$warn: mat-palette($mat-pink, 600, 500, 700);
 
 // Create the theme object (a Sass map containing all of the palettes).
-$web-ng-theme: mat-light-theme($web-ng-primary, $web-ng-accent, $web-ng-warn);
+$theme: mat-light-theme($primary, $accent, $warn);
+
+// Get generated palletes from theme. Sub-palettes listed in:
+// https://github.com/angular/components/blob/16f5f793dea3d83a6f08de14e3787ce97d82ff15/src/material/core/theming/_palette.scss#L674
+$foreground: map-get($theme, foreground);
```

Comment:
is foreground passed in as parameter something inbuilt we can use?

Reply:
`foreground` arg is the name of the palette we want to extract.